docs/whatsnew/index.rst

```diff
@@ -0,0 +1,11 @@
+What's new
+==========
+
+This section will present some highlights of new functionality integrated into Pylake.
+For a full list of new features and changes, please refer to the :doc:`changelog</changelog>`.
+
+.. toctree::
+    :caption: Contents
+    :maxdepth: 1
+
+    1.0.0/1_0_0
```

Comment:
not sure how I feel about the `.` in directory names. Maybe I'm just old school 😬